cnxepub/tests/test_collation.py

```diff
@@ -239,3 +239,45 @@ def test_without_ruleset(self):
 
         result = self.target(binder)
         self.assertIs(binder, result)
+
+    def test_with_ruleset(self):
+        binder = self.make_binder(
+            '8d75ea29',
+            metadata={'version': '3', 'title': "Book One"},
+            nodes=[
+                self.make_document(
+                    id="e78d4f90",
+                    content=b"<span>document one</span>",
+                    metadata={'version': '3',
+                              'title': "Document One"}),
+                self.make_document(
+                    id="3c448dc6",
+                    content=b"<span>document two</span>",
+                    metadata={'version': '1',
+                              'title': "Document Two"})])
+
+        # Append a ruleset to the binder.
+        ruleset = io.BytesIO(b"""\
+div[data-type='metadata']:last-of-type {
```

Comment:
Yeah, you're right. You can see this in the test, since 'Document One' is the value for title.